.cci.jenkinsfile

```diff
@@ -0,0 +1,36 @@
+// See https://github.com/jlebon/coreos-ci
+// This was originally copied from the rpm-ostree one
+@Library('github.com/coreos/coreos-ci-lib@master') _
+
+COSA_IMAGE = 'quay.io/coreos-assembler/coreos-assembler:latest'
+COSA_BUILDROOT_IMAGE = 'registry.svc.ci.openshift.org/coreos/cosa-buildroot:latest'
+
+stage("Build") {
+parallel rpms: {
```

Comment:
Maybe just `build`?

Reply:
Heh I saw that and was going to fix it but didn't want to wait for a whole other cycle of CI runs...